spec/factories/loans.rb

```diff
@@ -55,11 +55,12 @@
 
 
     trait :active do
-      status_option_id Loan::STATUS_OPTIONS.value_for('Active')
+      # note: can't seem to count on the seed.db data having been loaded when these are executed
+      status_option_id 1 #Loan.status_option_set.value_for_label('Active')
```

Comment:
Yes, relying on seeds ad hoc is also bad practice. 

The idea of using the above pattern as opposed to `find_or_create_by` is that it leverages the factories, so you don't have to specify e.g. division every time. `set_label_list` could also be invoked in the factory. The options factory could probably be modified to create an option set for itself if an appropriate one doesn't already exist using a similar method in the optionsets factory—it probably should anyway. And I was thinking you could pass 'active' as the argument to the `get_loan_status` method when that's the one you need. Does that make sense?